src/components/attendees/AttendeeRow.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './styles.css';
+import { Attendee } from '../../interfaces/Attendee';
+import { ReactComponent as OpenIcon } from '../../assets/Attendees/open.svg';
+import { ReactComponent as CheckedIcon } from '../../assets/Attendees/checked.svg';
+import { ReactComponent as UncheckedIcon } from '../../assets/Attendees/unchecked.svg';
+import { ReactComponent as EmailIcon } from '../../assets/Attendees/google.svg';
+import { ReactComponent as GoogleIcon } from '../../assets/Attendees/email.svg';
+
+/**
+ * @returns row for an attendee in the attendee table.
+ */
+
+interface AttendeeObject {
+	attendee: Attendee;
+}
+export default function AttendeeRow({ attendee }: AttendeeObject) {
```

Comment:
Can we just pass this in as an Attendee rather than packaging it then immediately unpackaging it?

Reply:
Unfortunately I think not.